src/com/esotericsoftware/kryo/Kryo.java

```diff
@@ -237,6 +239,13 @@ public Kryo (ClassResolver classResolver, ReferenceResolver referenceResolver) {
 		register(short.class, new ShortSerializer());
 		register(long.class, new LongSerializer());
 		register(double.class, new DoubleSerializer());
+
+		// Add RecordSerializer if JDK 14+ available
+		if (isClassAvailable("java.lang.Record")) {

```

Comment:
Please move this block directly below `ImmutableCollectionsSerializers.addDefaultSerializers(this);`.

Reply:
Done